### ISSUE TITLE
--fix-doc-comments not quite stable

### ISSUE THREAD

**Issue by davidmorgan** [dart-lang/dart_style#768]
Given this input

```
class Test {
  int x;
  /**
   * Hi.
   */
  int y;
}
```

`dartfmt --fix-doc-comments test.dart` produces this output

```
class Test {
  int x;
  /// Hi.
  int y;
}
```

but we're not done yet, as running _this_ through dartfmt produces

```
class Test {
  int x;

  /// Hi.
  int y;
}
```

**Comment by munificent** [comment]
Drat, that is a bug.

**Comment by munificent** [comment]
Closing since we're getting rid of `dart format --fix` in favor of `dart fix`: #1153
